src/helpers/isReactElementVisibleToAccessibility.ts

```diff
@@ -0,0 +1,40 @@
+import { ReactTestInstance } from 'react-test-renderer';
+
+export function isReactElementVisibleToAccessibility(
```

Comment:
Seems like this might be expensive to compute, as `findAll` does top-down tree travelsal while this function is doing bottom-up travelsal to root on each parent. Maybe we should have a `findAll` that would exclude component tree branches at it performs top-down travelsal. We could start with `findAll` code, duplicate it and add our tweak there.

Reply:
oh I thought you said the opposite on the previous PR that was opened

Reply:
<img width="918" alt="CleanShot 2022-08-16 at 15 56 09@2x" src="https://user-images.githubusercontent.com/32499425/184897649-fec31f65-c49d-481c-a022-0ee601dc0b26.png">

Reply:
Maybe we should try both approaches and compare performances?

Reply:
Haha, nice catch 💯 
Recently I've learnt that some of our users have pretty large test bases that take longer time to run, so I try to avoid added too much additional run time for them. I agree that measurement is the best way of deciding which way to go. We could start with the simplest (which seems to be traversing up to check parents) and then have another PR with faster algorithm if it's indeed faster in any way.

Reply:
Totally agree with your idea! Seems like the most pragmatic to me. And if people find it too slow at first for their codebase, they can still just not activate the option until we push a more optimised version

As for slow test, I've seen many project at my company that had big codebases and slow tests with RNTL and jesr. We even had to make a guide on how to improve their speed. Our most used quick win is using [sucrase](https://github.com/alangpierce/sucrase) for our tests now, which drastically improves the speed of our tests